bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -514,8 +517,8 @@ private String contributorLink(Contributor contributor) {
         return ret.toString();
     }
 
-    private String getChecksList(PullRequestCheckIssueVisitor visitor, boolean isCleanBackport, Map<String, Boolean> additionalProgresses) {
-        var checks = isCleanBackport ? visitor.getReadyForReviewChecks() : visitor.getChecks();
+    private String getChecksList(PullRequestCheckIssueVisitor visitor, boolean noNeedReview, Map<String, Boolean> additionalProgresses) {
```

Comment:
Can we flip this boolean to avoid negation in the name? I would suggest `reviewNeeded`.

Reply:
Sure, I will fix it.